_sections/100-encryption.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Encryption
+page: encryption
+---
+
+### Introduction
+
+This specification is a sub-specification of the [ODK XForms Specification](/). It describes how to encrypt an XForms submission in an manner that is compatible with ODK tools for data aggregation and decryption.
+
+### Overview
+
+A regular XForms submission consists of 1 XML file with optional additional media files from `<upload>` form controls. Forms that [have encryption enabled](/#encryption), will encrypt all these files and add an additional [submission manifest](#submission-manifest) XML file. The submission of these files occurs according to the [OpenRosa Form Submission Specification](https://docs.opendatakit.org/openrosa-form-submission/#form-submission), where the original XML file (now encrypted) is treated as a media file, and the submission manifest is treated as the XML file (aka the XForm Part).
+
+### Submission Manifest
+
+Below is an example of a valid submission manifest:
+
+```xml
+<data xmlns="http://opendatakit.org/submissions" id="mysurvey" encrypted="yes" version="2014083101">
+    <base64EncryptedKey>sHXUut13/res3S3uJkwgfhABOc74aXGnCTxTcRTplS9kflomxAzK35zcLc0BJu/Dro7FpPia4qU+f3yb3roJi/EUtRkTaHauAYDEX2OHZ4QThoSmbR0NJRw6kLjfkNS5bFaONWEbRn8eSbT7uyOGyvx5ddL3IKIxzu9vGzJX+cMpKKUQsORaXNEL7lRns7tVen93OSlYhSQak/CbAbkpsSpIW+Q13zrGv3n20YOHaun5yhSyZq6LeaHzPWKQv2POyl+N2j3NGbkz+RIvaVBLvTae4zB0iXlfTkYK9HwOKKDS6MI7z4g4L988WlQurkw5jlN5X9ahNhwZN2yLWTsnCQ==</base64EncryptedKey>
+    <encryptedXmlFile>submission.xml.enc</encryptedXmlFile>
+    <media>
+        <file>myimage.jpg.enc</file>
+        <file>myaudio.mp3</file>
```

Comment:
This would have a .enc extension, right?

Do we want to show this compact syntax even though Aggregate seems to botch it? I think it's ok because there should be an impending fix (https://github.com/opendatakit/aggregate/issues/319) but wanted to bring it up.

Reply:
I'd +1 to have the syntax Collect produces, which is the same one Aggregate generates:
```xml
<media>
  <file>blah.jpg.enc</file>
</media>
<media>
  <file>bleh.jpg.enc</file>
</media>
```

Reply:
Can do. I forgot to highlight this to you. My thought was, once both syntaxes are supported in Aggregate, to document only the nice one (and phase out the old one from Collect and Enketo in a few years). However, no big deal. We could update it later.